.travis.yml

```diff
@@ -8,7 +8,7 @@ compiler:
   - gcc
 
 env:
-  - PATH="/usr/local/opt/qt:$PATH"
+  - PATH="/usr/local/opt/qt:/usr/local/opt/qt/bin:$PATH"
```

Comment:
do we really need both opt/qt AND opt/qt/bin?

Reply:
@mpreisler fixed